maintainers-reviewers/RELEASE.MD

```diff
@@ -1,35 +1,23 @@
 # How to build and release projects on the Camunda Community Hub
-- [Maven](#maven)
-- [Docker](#docker)
-- [Onboarding your repository](#onboarding-your-repository)
-- [Maven](#maven-1)
-  - [Use the Camunda Community Hub groupId](#use-the-camunda-community-hub-groupid)
-  - [Add release parent to POM](#add-release-parent-to-pom)
-  - [Add GitHub workflow](#add-github-workflow)
-  - [Performing a release](#performing-a-release)
-- [Docker](#docker-1)
-  - [What if my release fails while deploying integration tests?](#what-if-my-release-fails-while-deploying-integration-tests)
-  - [What if my release says it's missing a key for GPG?](#what-if-my-release-says-its-missing-a-key-for-gpg)
-- [Integrating GitHub Actions with Zeebe on Camunda 8](#integrating-github-actions-with-zeebe-on-camunda-8)
-- [Troubleshooting](#troubleshooting)
```

Comment:
GitHub's Markdown viewer already provides an outline of all headlines, no need for duplication imho.